mammoth/conversion.py

```diff
@@ -179,8 +179,12 @@ def visit_table_cell(self, table_cell):
     
     
     def visit_line_break(self, line_break):
-        return [html.self_closing_element("br")]
-    
+        if line_break.line_break_type:
+            return [html.self_closing_element("br",{"class": line_break.line_break_type})]
+        else:
+            return [html.self_closing_element("br")]
```

Comment:
So far as I can tell, this doesn't use any style mappings. Specifically, it will unconditionally turn all breaks into `br` elements. I think we want to keep the default behaviour unchanged i.e. changing line breaks to `br` elements and ignoring other breaks, and then allowing this behaviour to be customised with style maps.